src/transform.jl

```diff
@@ -354,9 +354,9 @@ logpdf_with_trans(d::PDMatDistribution, x::Array{T,2}, transform::Bool) where {T
     U = cholesky(x).U
     n = dim(d)
     for i in 1:n
-      lp += (n - i + T(2)) * log(U[i, i])
+      lp += (n - i + 2.0) * log(U[i, i])
```

Comment:
Flux.TrackedReal doesn't have this construction.

Reply:
Sure, but
```julia
x = Flux.Tracker.TrackedReal(5.0)
y = x * T(2)
```
makes `y` a `TrackedReal`. The point is that constants don't need to be `Tracked`, but can interact with `Tracked` quantities.

Reply:
Maybe a callback function for TrackedReal would be OK?

Reply:
Ah, yes, you're completely correct. @MikeInnes any thoughts on best practice here?

Reply:
I think it would be preferably to avoid adding any `TrackedReal`-specific definitions. If there's nothing easy to do, then the current implementation is probably fine.

Reply:
Is use of `Union` type stable?

Reply:
In what context?

Reply:
I just pushed a solution to this. Please check.